specs/groupvolumesnapshots/cassandra-e2e/cassandra-restore-pvcs.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
```

Comment:
specs is for the deployments. Can you move these under /examples so that they are easier to find.

Reply:
moved